ext/drreg/drreg.h

```diff
@@ -117,9 +117,12 @@ typedef struct _drreg_options_t {
      * requested from DR via dr_raw_tls_calloc().  Any slots needed beyond
      * this number will use DR's base slots, which are not allowed to be
      * used across application instructions.  DR's slots are also more
-     * expensive to access (beyond the first few).
-     * This number should be computed as one plus the number of
-     * simultaneously used general-purpose register spill slots, as
+     * expensive to access (beyond the first few). DR's base slots may
+     * also be used by core DR or other clients, which may cause
```

Comment:
Core DR can be removed here: that's the source of the "not allowed to be used across application instructions"; beyond that there will be no conflict.

Reply:
For "other clients": I would stress that the problem is when using a client or library that doesn't use drreg and instead directly using DR register spill slots (dr_save_reg, etc.); i.e., all cooperating client components should be using drreg.

Reply:
Done.